ra/ra.go

```diff
@@ -429,6 +431,27 @@ func (ra *RegistrationAuthorityImpl) checkPendingAuthorizationLimit(ctx context.
 	return nil
 }
 
+func (ra *RegistrationAuthorityImpl) checkPendingOrderLimit(ctx context.Context, regID int64) error {
+	limit := ra.rlPolicies.PendingOrdersPerAccount()
+	if limit.Enabled() {
+		count, err := ra.SA.CountPendingOrders(ctx, regID)
+		if err != nil {
+			return err
+		}
+		// Most rate limits have a key for overrides, but like the pending
+		// authorization limit there is no meaningful key here. Only registration ID
+		// overrides make sense.
+		noKey := ""
+		if count >= limit.GetThreshold(noKey, regID) {
+			ra.pendOrdersByRegIDStats.Inc("Exceeded", 1)
+			ra.log.Info(fmt.Sprintf("Rate limit exceeded, PendingOrdersByRegID, regID: %d", regID))
```

Comment:
I know we log like this in other places, but I think we should discontinue that practice. The rate limit event will be logged in the WFE when it returns 429, along with the account id.

Reply:
Fixed in 0be45a3